platform/stellar/api.go

```diff
@@ -40,34 +42,61 @@ func (p *Platform) CurrentBlockNumber() (int64, error) {
 
 func (p *Platform) GetBlockByNumber(num int64) (*blockatlas.Block, error) {
 	if srcBlock, err := p.client.GetBlockByNumber(num); err == nil {
-		block := NormalizeBlock(srcBlock, p.CoinIndex)
+		block := NormalizeBlock(srcBlock, *p)
 		return &block, nil
 	} else {
 		return nil, err
 	}
 }
 
-func NormalizeBlock(block *Block, nativeCoinIndex uint) blockatlas.Block {
+func NormalizeBlock(block *Block, p Platform) blockatlas.Block {
 	return blockatlas.Block{
 		ID:     block.Ledger.Id,
 		Number: block.Ledger.Sequence,
-		Txs:    NormalizePayments(block.Payments, nativeCoinIndex),
+		Txs:    NormalizePayments(block.Payments, p),
 	}
 }
 
-func NormalizePayments(payments []Payment, nativeCoinIndex uint) (txs []blockatlas.Tx) {
+func NormalizePayments(payments []Payment, p Platform) (txs []blockatlas.Tx) {
+	var wg sync.WaitGroup
+	tupleChan := make(chan struct {
+		Payment;
+		TrxHash
```

Comment:
Declare the struct before to avoid declare twice below
```
	type Tuple struct {
		Payment
		TrxHash
	}
	var wg sync.WaitGroup
	tupleChan := make(chan Tuple)
```